samples/client/petstore/groovy/bin/main/org/openapitools/api/PetApi.groovy

```diff
@@ -214,7 +214,7 @@ class PetApi {
         contentType = 'multipart/form-data';
         bodyParams = [:]
         bodyParams.put("additionalMetadata", additionalMetadata)
-        bodyParams.put("file", file)
+        bodyParams.put("file", _file)
```

Comment:
i don't understand why this change occurs.

maybe merging the latest master into your branch and regenerate the samples to see if this change goes away?